spec/models/profile_spec.rb

```diff
@@ -1,4 +1,35 @@
 require "rails_helper"
+require "open-uri"
 
 RSpec.describe Profile, type: :model do
+  describe "Associations" do
+
+    it "must belong to a user" do
+      profile = (build(:profile))
+      expect(profile.user).to be_valid
+    end
+    
+    it "has a profile picture" do
+      profile = (build(:profile))
+      file = URI.open("https://images.unsplash.com/photo-1613679074971-91fc27180061?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1887&q=80")
+      profile.photo.attach(io: file, filename: "avatar.jpg")
```

Comment:
nice solution to image testing, assuming it reflects what Cloudinary does as well. Found this to understand what you were doing, could refactor slightly: https://guides.rubyonrails.org/v7.0.0/active_storage_overview.html

Reply:
As a profile picture will be added after the creation of the profile do we want to enforce it in the unit test? or do you want to rename the test to "can have photo attached"?